gpii/node_modules/preferencesServer/src/preferencesServer.js

```diff
@@ -52,6 +52,52 @@ fluid.defaults("gpii.preferencesServer", {
             route: "/preferences/:gpiiKey",
             method: "put",
             type: "gpii.preferencesServer.put.handler"
+        },
+        // GPII-2966
+        prefsSafeGet: {
+            route: "/prefssafe/:prefsSafeId",
```

Comment:
"prefsSafe" for all these endpoints would be consistent with our usage everywhere else

Reply:
Bump comment

Reply:
updated